scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -31,18 +31,55 @@ ruleTester.run('protractor-practices', rule, {
     code:
     `action.click(elem);
     browser.action.sleep();`,
+  }, {
+    code: 'a.then;',
+  }, {
+    code: 'then.c;',
+  }, {
+    code: 'then().d;',
+  }, {
+    code: 'a.then.b;',
   }],
 
   invalid: [
     {
       code:
-      `it('should test a feature', function() {
+        `it('should test a feature', function() {
         browser.sleep();
       });`,
       errors: [{
         message: 'Please do not use browser.sleep() in protractor files',
         type: 'CallExpression',
       }],
     },
+    {
+      code: 'z.then();',
```

Comment:
Please use a real example so that the test code makes more sense, e.g: `someElement.click().then(() => {expect(...).toBe(...)})`

(Ditto below)

Reply:
Done! I changed test-code to look more realistic. Also removed some test-code which I thought were unnecessary. Does it look good to you?